components/actiongroup/stories/actiongroup.stories.js

```diff
@@ -69,14 +69,17 @@ Default.args = {
 	content: [
 		{
 			iconName: "Edit",
+			iconSet: "workflow",
```

Comment:
That one is passing through the ActionButton args through `content[]`. it's good to be specific and also set the icon set whenever the icon name is being set, though it's not necessarily always required; in this case it would fall back to the default value of "workflow".